pkg/work/spoke/options.go

```diff
@@ -28,6 +30,7 @@ func NewWorkloadAgentOptions() *WorkloadAgentOptions {
 		MaxJSONRawLength:                       1024,
 		StatusSyncInterval:                     10 * time.Second,
 		AppliedManifestWorkEvictionGracePeriod: 60 * time.Minute,
+		WorkloadSourceDriver:                   work.ConfigTypeKube,
```

Comment:
also need to default the kubcoonfig file path

Reply:
You're right, added.